integration/benchmark/cmd/ci/main.go

```diff
@@ -316,13 +316,16 @@ func adjustTPS(
 	lastTs := time.Now()
 	lastTPS := float64(minTPS)
 	lastTxs := uint(lg.GetTxExecuted())
+	lastTimedoutTxs := lg.GetTxTimedout()
 	for {
 		select {
 		// NOTE: not using a ticker here since adjusting worker count in SetTPS
 		// can take a while and lead to uneven feedback intervals.
 		case nowTs := <-time.After(interval):
 			currentSentTxs := lg.GetTxSent()
 			currentTxs := uint(lg.GetTxExecuted())
+			currentTimedoutTxs := lg.GetTxTimedout()
```

Comment:
can you add descriptions here explaining what the vars represent? For example, what's the difference between `currentTxs` and `currentTimedoutTxs`?